app/controllers/job/tasklist.js

```diff
@@ -1,14 +1,19 @@
 import Controller from '@ember/controller';
 import { tracked } from '@glimmer/tracking';
-import { task } from 'ember-concurrency-decorators';
 import { service } from '@ember/service';
+import { task } from 'ember-concurrency-decorators';
 
-export default class JobsDetailsController extends Controller {
+export default class JobTasklistController extends Controller {
   @service router;
 
   @tracked page = 0;
   @tracked sort = 'created';
   @tracked size = 15;
+  @tracked job;
+
+  sort = '-index';
```

Comment:
I agree that for a typical end user the index is almost useless, but so is the URI of the Task and the full URI of the operation. These could be nicely formatted strings for example. However, this application is meant for technical people, and thus a bit of redundant information is no big deal. Also, it might help debugging.